.ci/jobs/apm-agent-nodejs-mbp.yml

```diff
@@ -26,6 +26,9 @@
         - regular-branches: true
         - change-request:
             ignore-target-only-changes: false
+        - named-branches:
+            - wildcards-name:
+                excludes: 'greenkeeper*'
```

Comment:
this will build only the PR from greenkeeper, right now is building the branch and the PR.